packages/bcer-retailer-app/app/src/components/MyBusiness/MyBusinessSubmission.tsx

```diff
@@ -1,281 +1,281 @@
+import { makeStyles, Typography, Paper } from '@material-ui/core';
+import { StyledButton, StyledTable } from 'vaping-regulation-shared-components';
+import { useAxiosGet } from '@/hooks/axios';
+import { CSVLink } from 'react-csv';
+import SaveAltIcon from '@material-ui/icons/SaveAlt'
+
+import { SuccessStepEnum, BusinessLocationHeaders } from '@/constants/localEnums';
+import { IBusinessLocationValues } from '@/components/form/validations/vBusinessLocation';
+import { BusinessDetails } from '@/constants/localInterfaces';
+import SuccessStep from '@/components/successStep/SuccessStep'
+import FileCheckGreen from '@/assets/images/file-check-green.png';
+import { AppGlobalContext } from '@/contexts/AppGlobal';
+import { formatError } from '@/utils/formatting';
+
+const useStyles = makeStyles({
+  title: {
+    color: '#0F327F',
+    paddingBottom: '30px'
+  },
+  box: {
+    border: 'solid 1px #CDCED2',
+    borderRadius: '4px',
+    padding: '1.4rem 1.4rem 0rem 1.4rem',
+    marginBottom: '30px'
+  },
+  successBanner: {
+    display: 'flex',
+    backgroundColor: '#E7F9EA',
+    borderRadius: '4px',
+    padding: '20px',
+    alignItems: 'center',
+    marginBottom: '20px',
+  },
+  bannerIcon: {
+    height: '45px',
+    paddingRight: '20px'
+  },
+  bannerText: {
+    fontWeight: 600,
+    color: '#3A3A3A'
+  },
+  description: {
+    color: '#565656',
+    paddingBottom: '20px'
+  },
+  stepsSubtitle: {
+    paddingBottom: '20px'
+  },
+  boxTitle: {
+    fontSize: '17px',
+    fontWeight: 600,
+    lineHeight: '22px',
+  },
+  boxHeader: {
+    fontSize: '17px',
+    fontWeight: 600,
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    paddingBottom: '15px',
+  },
+  actionsWrapper: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    paddingBottom: '10px'
+  },
+  boxDescription: {
+    fontSize: '14px',
+    color: '#3A3A3A',
+    lineHeight: '20px',
+    paddingBottom: '15px',
+  },
+  boxRow: {
+    display: 'flex',
+    paddingBottom: '20px',
+  },
+  buttonIcon: {
+    paddingRight: '5px',
+    color: '#285CBC',
+    fontSize: '20px',
+  },
+  csvLink: {
+    textDecoration: 'none',
+  },
+  rowTitle: {
+    fontSize: '14px',
+    color: '#424242',
+    width: '300px'
+  },
+  rowContent: {
+    fontSize: '14px',
+    fontWeight: 600,
+    color: '#3A3A3A',
+  },
+  editButton: {
+    fontSize: '14px',
+    width: '150px',
+    minWidth: '150px'
+  },
+  tableWrapper: {
+    marginBottom: '20px'
+  }
+})
+
+
+export default function MyBusinessSubmission (props: any) {
+  const classes = useStyles();
+
+  const [{ data: submission, error }, get] = useAxiosGet(`/submission/${props.match.params.submissionId}`);
+  const [details, setDetails] = useState<BusinessDetails>();
+  const [locations, setlocations] = useState<Array<IBusinessLocationValues>>();
+  const [appGlobal, setAppGlobalContext ] = useContext(AppGlobalContext);
+
+  useEffect(() => {
+    if (submission) {

```

Comment:
Not related to pr. but this check doesn't really make much sense to me. Maybe we should create a ticket to address this later?

Reply:
completely agree, this doesn't seem to make much sense but looks like it's there for a reason. we can definitely investigate what its doing and see if its necessary